### PR TITLE
fixed eap8 path and wf27

### PR DESCRIPTION
**Pull request by olkornii**
Signed-off-by: Oleksii Korniienko <olkornii@redhat.com>

### CODE REVIEW

**Comment by olkornii** [comment]
@robstryker check please.

File: stacks.yaml [hunk 1]
```diff
@@ -3345,7 +3345,7 @@ availableRuntimes:
    version: 8.0.0
    license: *lgpl21
    url: https://developers.redhat.com/products/eap
-   downloadUrl: http://download.hosts.prod.upshift.rdu2.redhat.com/devel/candidates/middleware/eap/JBEAP-8.0.0.Beta-CR1/jboss-eap-8.0.0.Beta-CR1.zip
+   downloadUrl: http://download.hosts.prod.upshift.rdu2.redhat.com/devel/candidates/middleware/eap/JBEAP-8.0.0-Beta/jboss-eap-8.0.0.Beta.zip
```
Comment (jeffmaury) [inline stacks.yaml]: Is EAP8 Beta public ?

Comment (olkornii) [reply stacks.yaml]: As I know - NO, thats why we have [unreleased-runtimes](https://github.com/jboss-developer/jboss-stacks/tree/unreleased-runtimes) branch

Comment (olkornii) [reply stacks.yaml]: Or what do you mean "public"?

Comment (jeffmaury) [reply stacks.yaml]: Accessible by anyone

Comment (jeffmaury) [reply stacks.yaml]: outside Red Hat

**Pull request closed without merging**